doc/safety.md

```diff
@@ -0,0 +1,61 @@
+# SECURITY
+
+This document describes the security recommendations and issues that can be helpful for users of Bitcoin Core.
+
+
+Download Bitcoin Core only from below links, verify the integrity of the download before using:
+
+https://bitcoincore.org/en/download/
+
+https://github.com/bitcoin/bitcoin/releases
+
+Downloading Bitcoin Core from untrusted sources can result in using a [malware](https://bitcoin.stackexchange.com/a/107738/) that can steal your bitcoin and compromise your computer.
+
+### P2P
+
+DNS seeds are used for bootstrapping the P2P network. If one of the domains used for default DNS seeds is hacked, this can impact nodes. A node checks 3 seeds at a time, so it would require coordination for a malicious seed to affect the network. If you don't believe the DNS seeds hardcoded in Bitcoin Core, you should manually make connections to nodes you do trust (using `-seednode` / `-addnode`). You can also disable the DNS seeds (`-dnsseed=0`) in that case.
+
+Difference in DNS seed and Seed node: https://bitcoin.stackexchange.com/a/17410/
+
+Having multiple connections to the network is fundamental to safety against eclipse attacks. If a user configured the node to only have outbound block-relay-only connections, they wouldn't be participating in addr relay, so wouldn't have a diverse addrman to identify candidates to connect to. It is suggested to have at least 10 outbound connections.
```

Comment:
Suggest using either the second person ("you") or third person ("a user") consistently throughout the document.

 ```suggestion
Having multiple connections to the network is fundamental to safety against eclipse attacks. If you configure your node to only have outbound block-relay-only connections, they wouldn't be participating in addr relay, so wouldn't have a diverse addrman to identify candidates to connect to. It is suggested to have at least 10 outbound connections.
```

Reply:
Done in `088d99b5e5c9bfd907861ce68692e638ee56448d`